### ISSUE TITLE
English error - lend vs borrow

### ISSUE THREAD

**Issue by kc1212** [RalfJung/rust-101#8]
In [here](https://github.com/RalfJung/rust-101/blob/master/src/part04.rs#L53) and [here](https://github.com/RalfJung/rust-101/blob/master/src/part04.rs#L55), the correct way is to use "lend", i.e. "If you _lend_ a book to your friend...".  You may want to rephrase it to either that or something like "If your friend borrows a book from you...".

**Comment by RalfJung** [comment]
That choice of terminology for Rust keeps giving me a hard time writing.^^

This [dictionary](https://www.dict.cc/?s=borrow) claims that "to borrow" is a valid verb in English. What's wrong with the way I am using it?

**Comment by kc1212** [comment]
In your context it should be either "borrow from" or "lend to", so "borrow to" wouldn't be right. "to borrow" can be used in some other context, something like "This book is for Alice to borrow". I'm not an English major so I won't attempt to explain it but more information can be found [here](http://www.eslcafe.com/grammar/confusing_words_lend_borrow.html) and [here](https://dictionary.cambridge.org/grammar/british-grammar/lend-or-borrow).

**Comment by RalfJung** [comment]
Fixed by https://github.com/RalfJung/rust-101/commit/a3a64118702b4f75691de78d42256c306f286014 . Thanks!
